internal/pkg/cli/interfaces.go

```diff
@@ -133,11 +133,20 @@ type ecrService interface {
 	GetECRAuth() (ecr.Auth, error)
 }
 
+type vpcService interface {
```

Comment:
That makes sense since all of the vpc methods are from the `ec2` package. My concern is that the interface is specifically declared for vpc-related methods (subnets, security-ids), while ec2 is not necessarily limited to vpc functionalities. Also from [AWS documentation](https://docs.aws.amazon.com/) I feel like they are trying to make a clear separation between EC2 and VPC (they are in completely different categories: Compute and Networking)